.gitignore

```diff
@@ -14,3 +14,6 @@ logs/*
 
 # Pyenv
 .python-version
+
+#to avoid conflict
+project.db
```

Comment:
This doesnt solve anything there is still a conflict. Also we want to not just ignore project.db, you can ignore it locally however with .git/info/exclude

Reply:
ahh okay
sorry my bad